### PR TITLE
[NEW] initial commit with code of conduct

### PR DESCRIPTION
**Pull request by EdwardAngert**
XR #1375

### CODE REVIEW

**Comment by GuessWhoSamFoo** [comment]
Also need to update tests to ignore code of conduct for filename rules.

File: docs/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+## Our Responsibilities
+
+As project maintainers, we are responsible for clarifying the standards of acceptable behavior and we are expected to take appropriate and fair corrective action in response to any instances of unacceptable behaviour.
+
+We have the right and responsibility to remove, edit, or reject:
+
+* comments
+* commits
+* code
+* wiki edits
```
Comment (Guaris) [inline docs/CODE_OF_CONDUCT.md]: No Wiki Consider removing

File: docs/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+* posting via an official social media account
+* participating in an online or offline event
+
+Project maintainers may further define and clarify representation of a project.
+
+## Enforcement
+
+You should report any instances of abusive, harassing, or otherwise unacceptable behaviour to the project team at docs@linode.com. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain the anonymity of the reporter of an incident. We may post further details of specific enforcement policies separately.
+
+Project maintainers who do not follow or enforce this code of conduct in good faith may face temporary or permanent consequences. These will be determined by members of the project's leadership.
```
Comment (Guaris) [inline docs/CODE_OF_CONDUCT.md]: Change wording here to: project contributors. No one maintains this but us.

File: docs/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (EdwardAngert) [reply docs/CODE_OF_CONDUCT.md]: @Guaris would we want to keep it as a "if ever we add a wiki, this applies there too," or would you rather we just re-edit when/if ever we add a wiki?

**Comment by GuessWhoSamFoo** [comment]
Add filename here: https://github.com/linode/docs/blob/b327f9fdac26c80f3a6ed3ab9c524964fd9d912f/ci/test_filename.py#L7

**Comment by EdwardAngert** [comment]
@GuessWhoSamFoo https://github.com/linode/docs/pull/1423

**Pull request merged**